File: WaterManager/src/main/java/pissir/watermanager/model/item/Approvazione.java
package pissir.watermanager.model.item;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

/**
 * @author Almasio Luca
 * @author Borova Dritan
 * @author Gattico Alessandro
 */

@Setter
@Getter
@NoArgsConstructor
public class Approvazione {
	
	private int id;
	private int idRichiesta;
	private int idGestore;
	private boolean approvato;
	private String date;
	
	
	public Approvazione(int id, int idRichiesta, int idGestore, boolean approvato,
						String date) {
		this.id = id;
		this.idRichiesta = idRichiesta;
		this.idGestore = idGestore;
		this.approvato = approvato;
		this.date = date;
	}
	
}

File: WaterManagerUI/Model/Item/Approvazione.cs
namespace WaterManagerUI.Model.Item;

public class Approvazione
{
    public int id { get; set; }
    public int idRichiesta { get; set; }
    public int idGestore { get; set; }
    public bool approvato { get; set; }
    public String date { get; set; }

    public Approvazione()
    {
    }

    public Approvazione(int id, int idRichiesta, int idGestore, bool approvato, string date)
    {
        this.id = id;
        this.idRichiesta = idRichiesta;
        this.idGestore = idGestore;
        this.approvato = approvato;
        this.date = date;
    }
}